doit/action.py

```diff
@@ -214,15 +214,17 @@ def execute(self, out=None, err=None):
 
 
     def expand_action(self):
-        """expand action string using task meta informations
-        @returns (string) - expanded string after substitution
+        """Expand action using task meta informations if action is a string.
+        Convert `Path` elements to `str` if action is a list.
```

Comment:
I will add validation and raising `InvalidTask` if the user provides something that is nor `str` nor `Path`. I mentioned it earlier: https://github.com/pydoit/doit/pull/114#issuecomment-157982283